tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -251,6 +251,12 @@ describe(__filename, () => {
     expect(root.find('#collectionSlug')).toHaveProp('value', 'new-slug');
   });
 
+  it('has no search add-on selection when creating selection', () => {
```

Comment:
This should say `hides search add-on select when creating a collection`.

Reply:
Ok, thanks!
It looks better!